infinispan-remote/src/main/java/org/hibernate/ogm/datastore/infinispanremote/impl/InfinispanRemoteStoredProceduresManager.java

```diff
@@ -64,16 +61,9 @@ private Object execute( RemoteCache<Object, Object> executor, String storedProce
 			String msg = e.getMessage();
 			if ( e instanceof HotRodClientException && msg != null ) {
 				// parse undefined task exception, e.g. ISPN027002: Unknown task 'storedProcedureName'"
-				if ( msg.contains( "ISPN027002" ) ) {
+				if ( msg.contains( UNKNOWN_TASK_ERROR_ID ) ) {
 					throw log.procedureWithResolvedNameDoesNotExist( storedProcedureName, e );
 				}
-				Matcher matcher = REFERENCE_ERROR_REGEXP.matcher( msg );
-				if ( matcher.find() ) {
-					String param = matcher.group( 1 );
-					if ( param != null && !param.isEmpty() ) {
-						throw log.cannotSetStoredProcedureParameter( storedProcedureName, param, namedParameters.get( param ), e );
-					}
-				}
```

Comment:
I would assume that this case has a specific error code. isn't it?

Reply:
Yes. But I and @fax4ever think that it is excess.

Reply:
Don't we have a test for this?

Reply:
@DavideD , yes, we have a test for it. And it's failing because it wants `invalid parameters` but got `cannot execute` exception. Should we ignore it?

Reply:
I wouldn't skip it, but change so that it can catch the different exception. Does the error message should contain some information about what caused the error?